types/VariableTypes.ts

```diff
@@ -14,4 +14,10 @@ export type Variable  = {
     occurrences?: number;
     value?: string;
     defaultValue?: string;
+}
+
+export type MoveVariable = {
+    id: string;
+    parent: string;
+    order: number;
 }
```

Comment:
shouldnt this EOL be handled by prettier and the pre-commit hook?

Reply:
Nope, we have endOfLine set to auto, which means maintaining existing line endings